### PR TITLE
feat: [IDP-2063]: Initial draft for custom dashboard docs

### PR DESCRIPTION
**Pull request by Debanitrkl**
# Harness Developer Pull Request
Initial draft for custom dashboard docs

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

**Review by Valerra9201 — CHANGES_REQUESTED** [review]
deleted the spam content

**Review by Valerra9201 — COMMENTED** [review]
[Backup-codes-zabiyakovaleriy6.txt](https://github.com/harness/developer-hub/files/14304678/Backup-codes-zabiyakovaleriy6.txt)

File: docs/internal-developer-portal/get-started/custom-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: Custom Dashboards
+description: Using custom dashboards to enhance developer experience
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/get-started/custom-dashboards.md:3]:
```suggestion
description: Using Custom Dashboards to measure key IDP metrics around Software Catalog and Workflows
```

File: docs/internal-developer-portal/get-started/custom-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+sidebar_position: 120
+---
+
+## Introduction
+
+Custom Dashboards are a Harness Platform Feature, that allows you to key metrics and data related to your module. You can customize the dashboards to show specific metrics and data relevant to your needs, and access them from a central location. The Harness Dashboards provide actionable insights out of your IDP, so you can enable good developer experience in your org.
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/get-started/custom-dashboards.md:9]:
```suggestion
Dashboards is a Harness Platform Feature, that allows you to visualize key metrics and data related to your module. You can customize the dashboards to show specific metrics and data relevant to your needs, and access them from a central location. The Harness Dashboards provide actionable insights out of your IDP, so you can enable good developer experience in your org. Dashboards also allow you to setup scheduled delivery of reporting and alerts for your executives and stakeholders.
```

File: docs/internal-developer-portal/get-started/custom-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- Make sure you have the `Manage` permissions for **Dashboards** to create, edit, and delete custom dashboards.
+- Make sure you have the `View` permissions for **Dashboards** to view the custom dashboards.
+
+:::
+
+## Navigate to Dashboards
+
+To view and access dashboards, select **DASHBOARDS** from the left navigation in your Harness account. You can filter the data on the dashboards for IDP module by selecting the module name from the top of the page. 
+
+#### Present Left Nav
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/get-started/custom-dashboards.md:23]:
```suggestion
#### Old Left Nav
```

Think it's called Old now

File: docs/internal-developer-portal/get-started/custom-dashboards.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+3. Dashboard with overview of all the scorecards and their respective checks. 
+
+![](./static/scorecards-dashboard.png)
+
+4. Dashboard describing the Workflow Execution details
+
+![](./static/workflows-dashboard.png)
+
+
+## Create Dashboard
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/get-started/custom-dashboards.md:52]: IMO showing an example of at least one IDP dashboard here will be useful.

**Review by OrkoHunter — APPROVED** [review]

File: docs/internal-developer-portal/get-started/custom-dashboards.md (hunk 4 above)
Comment (Debanitrkl) [reply docs/internal-developer-portal/get-started/custom-dashboards.md:52]: We can add examples it once it's available now it's just limited to looker. Now we can't add any dashboards.

**Review by sathish-soundarapandian — APPROVED** [review]

**Pull request merged**